frontend/test/pytest/test_mid_circuit_measurement.py

```diff
@@ -210,6 +212,256 @@ def circuit(x):
         assert circuit(0.0) == [0] * 10
         assert circuit(jnp.pi) == [1] * 10
 
+    def test_dynamic_one_shot_with_sample_single(self, backend):
```

Comment:
Have a look at the test just above this one and see if it works now :) (it's currently marked xfail)

Looking at it it's actually very similar to this test

Reply:
Correct, the above test works now.